common_clang.cpp

```diff
@@ -245,6 +245,7 @@ Compile(const char *pszProgramSource, const char **pInputHeaders,
     compiler->createFileManager(OverlayFS);
     compiler->createSourceManager(compiler->getFileManager());
 
+    llvm::cl::ResetAllOptionOccurrences();
```

Comment:
Could you please put a comment here that this is a workaround for strange code in llvm.org?